packages/graphql-server-module-graphiql/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "graphql-server-module-graphiql",
-  "version": "0.4.4",
+  "version": "0.6.1",
```

Comment:
You have issues with the build because someone already broke it (the same way this does), #334 fixes just that.
So please revert this version change, and we will rebase and merge this